lib/src/Picker/Picker.tsx

```diff
@@ -134,8 +160,6 @@ export const Picker: React.FunctionComponent<PickerProps> = ({
       </div>
     </div>
   );
-};
+}
 
-Picker.defaultProps = {
-  ...datePickerDefaultProps,
-};
+Picker.displayName = 'Picker';
```

Comment:
What's the motivation for setting a display name? The current tradeoff we use in the main repository is to ensure a correct display name in development and to optimize for bundle size in production (a marginal gain).

Reply:
For some reason our components are missing displaynames in a lot of cases. I need to investigate the issue

Reply:
If it's related to forward ref, we have this solution:
```jsx
const Rating = React.forwardRef(function Rating(props, ref) {
```